crates/topos-tce-api/src/stream/mod.rs

```diff
@@ -155,7 +155,7 @@ impl Stream {
                 Some(stream_packet) = self.inbound_stream.next() => {
                     match stream_packet {
                         Ok((request_id, _message)) => {
-                            debug!("Received message for stream: {request_id:?}");
+                            debug!("Received message from stream_id: {request_id:?}");
```

Comment:
`stream_id: {request_id}`

My previous comment was that we read `... stream: {request_id}`, so I'm confused whether you want to print here `self.stream_id` or `request_id`

Reply:
Ah of course. I got confused. Yes, I wanted to print `self.stream_id` instead of the `request_id`.